pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/FunctionActioner.java

```diff
@@ -153,9 +153,13 @@ RuntimeSpawner getRuntimeSpawner(Function.Instance instance, String packageFile)
         int instanceId = instance.getInstanceId();
 
         FunctionDetails.Builder functionDetailsBuilder = FunctionDetails.newBuilder(functionMetaData.getFunctionDetails());
+        
```

Comment:
can we add a unit test for this and or some comments

Reply:
I can do that